nengo_loihi/loihi_cx.py

```diff
@@ -252,8 +252,9 @@ def discretize(target, value):
 
 
 class CxSynapses(object):
-    def __init__(self, n_axons):
+    def __init__(self, n_axons, label=None):
```

Comment:
I like adding the labels a lot, makes the code that creates `CxSynapses` more clear :+1: